html/store.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <title>Karma</title>
+    <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="/css/store.css">
+    <link rel="stylesheet" href="/css/style.css">
+    <link rel="shortcut icon" type="image/x-icon"  href="/favicon.ico">
+
+</head>  
+<header>
+    <img src="/img/karma-logo.svg" alt="karma-logo">
+    <nav id="nav" class="nav">
```

Comment:
```suggestion
    <nav id="nav" class="nav_container">
```
This is so person who reads your file understands your naming system a bit better. Look into BEM naming system :)